addins/Cake.FtpDeploy.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.FtpDeploy
 NuGet: Cake.FtpDeploy
 Assemblies:
 - "/**/Cake.FtpDeploy.dll"
-Repository: https://github.com/Supermortal/Cake.FtpDeploy
+Repository: https://www.nuget.org/packages/Cake.FtpDeploy/
```

Comment:
@Jericho I am concerned about this type of change.  What is it that causes this change?  Is it something missing from the NuGet metadata?  I would rather that this change "wasn't" made ,if it can be helped.

Reply:
Two reasons: the projectUrl is missing from nuspec AND this project hasn't been move to the cake-contrib. If the author adds the necessary URL in nuspec or if they move their project under the cake-contrib organization then AddinDiscoverer will be able to figure out the URL.

Reply:
To make matters more confusing: the [nuspec info in this project](https://github.com/Supermortal/Cake.FtpDeploy/blob/master/src/Cake.FtpDeploy/Cake.FtpDeploy.csproj#L15) does not match the nuspec in the published package!!?!?!?!